components/avatar.css

```diff
@@ -0,0 +1,45 @@
+.avatar-box,
+.avatar-circle {
+	margin: 1rem 1rem;
```

Comment:
You can use utility classes for spacing(if you have created) instead of declaring it here.